Dockerfile-prod

```diff
@@ -15,4 +15,7 @@ COPY src/ src/
 # Copying .env files
 COPY .env* ./
 
+# Copying babel config
+COPY .babelrc ./
+
```

Comment:
Shouldn't be necessary for prod, I'd say?

Reply:
The `sortBy?.length` would fail otherwise

Reply:
Nevermind, I read `eslint` instead of `babel` 🤦‍♂️